### PR TITLE
add pyfakefs to make tests independents of the file system

### PR DESCRIPTION
**Pull request by Gryfenfer97**

### CODE REVIEW

File: tests/test_path.py [hunk 1]
```diff
@@ -1,198 +1,198 @@
...
-def test_windows_is_common_executable_path_mechanical(
-    mock_is_valid_executable_path, path, expected
-):
-    assert _is_common_executable_path("mechanical", path) == expected
+def test_get_available_ansys_installation(mock_filesystem):
+    print(dict(zip([202, 211, 231], ANSYS_INSTALLATION_PATHS)))
+    print(get_available_ansys_installations())
+    assert get_available_ansys_installations() == dict(
+        zip([202, 211, 231], ANSYS_INSTALLATION_PATHS)
+    )
```
Comment (germa89) [inline tests/test_path.py:158]: Why the prints?

**Review by germa89 — APPROVED** [review]
It looks great to me. Good job @Gryfenfer97

**Comment by germa89** [comment]
Pinging @clatapie for learning purposes.

File: tests/test_path.py (hunk 1 above)
Comment (Gryfenfer97) [reply tests/test_path.py]: oops looks like I forgot to remove them

**Pull request merged**